job-dsls/jobs/srcclr_scan_job.groovy

```diff
@@ -8,7 +8,7 @@ folder("${srcclrPath}")
 
 for (repo in repoList) {
 
-    def jobName = "srcclr-scan-${repo}"
+    def jobName = "${srcclrPath}/srcclr-scan-${repo}"
```

Comment:
Just a very minor comment, maybe the constant _SRCCLR_JOBS_FOLDER_ could be used here directly and the variable _srcclrPath_ is not needed, as it is in [1]?

[1] https://github.com/kiegroup/kie-jenkins-scripts/blob/master/job-dsls/jobs/pr_jobs.groovy#L161